graph/articles/collections.md

```diff
@@ -316,3 +316,148 @@ Content-Type: application/json
    "value": []
 }
 ```
+
+## 11. Collections of structural types (complex types or primitive types)
+
+Collections of entity types are generally preferable to collections of structual types because collections of structural types must be updated as a single unit, meaning that they are overwritten entirely by new contents, rather than be updated relative to the existing contents. 
+Sometimes, structural collection properties are added to a type and then scenarios are discovered later that require a collection of entity types.
+Take the following model with an entity type `foo` that has a collection of `bar`s:
+
+```xml
+<EntityType Name="foo">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="bars" Type="Collection(self.bar)" />
+</EntityType>
+
+<ComplexType Name="bar">
+  <Property Name="prop1" Type="Edm.String" />
+  <Property Name="prop2" Type="Edm.String" />
+</ComplexType>
+```
+and a scenario arises that requires, for example, to remove individual `bar`s from the collection. 
+There are two options forward: //// TODO do we want to offer both options, or just one?
+
+### 11.1 Side-by-side collection properties
+
+The model can be updated to have two collections side-by-side:
+```diff
+<EntityType Name="foo">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="bars" Type="Collection(self.bar)" />
++ <NavigationProperty Name="barsAsEntities" Type="Collection(self.barAsEntity)" ContainsTarget="true" />
+</EntityType>
+
+<ComplexType Name="bar">
+  <Property Name="prop1" Type="Edm.String" />
+  <Property Name="prop2" Type="Edm.String" />
+</ComplexType>
+
++<EntityType Name="barAsEntity">
++ <Key>
++   <PropertyRef Name="prop1" />
++ </Key>
++ <Property Name="prop1" Type="Edm.String" />
++ <Property Name="prop2" Type="Edm.String" />
++</EntityType>
+```
+Clients will now be able to refer to individual `bar`s using `prop1` as a key, and they can now remove those `bar`s using `DELETE` requests:
+```http
+DELETE /foos/{fooId}/bars/{some_prop1}
+```
+```http
+HTTP/1.1 204 No Content
+```
+The expectation is that `bars` and `barsAsEntities` are treated as two "views" into the same data.
+To meet this expectation, workloads must:
+1. Keep the properties consistent between `bar` and `barAsEntity`.
```

Comment:
Do we really want to keep maintaining `bar`, or would it be better to deprecate `bar` and only add new properties to `barAsEntity`?

Reply:
At the very least, this section needs to be honored by workloads for the period that both properties exist. I think we are agreed on that?

I personally don't feel strongly about driving workloads to deprecate the old property. I think that their own supportability overhead will be a forcing function for them, and if they don't have significant overhead, I don't see a problem with having both properties. 

Maybe I should just add a sentence that says that the old property should be marked deprecated (and leave out whether it ever gets removed)?

Reply:
Agree that these semantics need to hold for the duration during which both versions of the property are supported. I think this is called out in the deprecation spec as well.  Happy to leave this here, and add that the old property should be marked as deprecated.

Reply:
Sounds good, the old property got marked deprecated in a previous iteration, so I'm resolving this comment